docs/DeveloperGuide.md

```diff
@@ -285,29 +285,71 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: UC01 - Delete a member**
+
+**Guarantees: MSS -> Specified member will be deleted.**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1. User requests to <ins>list all members</ins> (UC02)
+2. CCACommander shows a list of all members
+3. User requests to delete a specific member in the list of all members
+4. CCACommander deletes the specific member
 
     Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+* 2a. The list of all members is empty.
+
+  * 2a1. CCACommander shows an error message.
```

Comment:
Should this be CCAComander displays an empty list?

Reply:
Agreed, will make the 2 changes suggested by you.

Reply:
Yes, I think it can be removed from the other use cases as well. It should not be an extension because this is already covered under the MSS of the listing use case. Displaying an empty list is equivalent to displaying the entire list when the list is empty.